package.json

```diff
@@ -67,7 +67,7 @@
         "production": "npm start",
         "unlocalize": "grunt unlocalize",
         "test": "grunt test && grunt build-browser-compressed",
-        "start": "http-server -p 8000 --cors"
+        "start": "npm run localize && http-server -p 8000 --cors && npm run unlocalize"
```

Comment:
The `npm run unlocalize` script actually does run after `ctrl+c`. We need to do unlocalize to remove the changes to the `src/nls` folder when we do localize. Otherwise, contributors would be confused with all the changes that they didn't make.